python/ray/_private/function_manager.py

```diff
@@ -231,9 +254,17 @@ def get_execution_info(self, job_id, function_descriptor):
             # Load function from local code.
             # Currently, we don't support isolating code by jobs,
             # thus always set job ID to NIL here.
-            job_id = ray.JobID.nil()
+            job_id_local = ray.JobID.nil()
             if not function_descriptor.is_actor_method():
-                self._load_function_from_local(job_id, function_descriptor)
+                try:
+                    self._load_function_from_local(job_id_local,
```

Comment:
we should still let `_load_function_from_local` use `load_function_or_class_from_local`